pyhgvs/variants.py

```diff
@@ -173,13 +173,13 @@ def _on_forward_strand(self):
             seq_3p = self.seq_3p
             self.seq_5p = revcomp(seq_3p)
             self.seq_3p = revcomp(seq_5p)
-            self.alleles = map(revcomp, self.alleles)
+            self.alleles = [revcomp(allele) for allele in self.alleles]
 
     def _trim_common_prefix(self):
         """
         Trim the common prefix amongst all alleles.
         """
-        minlength = min(map(len, self.alleles))
+        minlength = min(list(map(len, self.alleles)))
```

Comment:
I think `list` was probably a holdover from running `2to3`, which tends to assume all iterables need to become sequence types.